src/components/RepoListItem.js

```diff
@@ -12,7 +13,7 @@ function RepoListItem({goal, stars}) {
         <FlexCenter>
           <Avatar alt="avatar" src={`https://avatars.githubusercontent.com/${goal.title.split("/")[0].replace(/\s+/g, "")}`} />
           <FlexColumn className="details">
-            <p>{goal.title.replace(/\s+/g, "")}</p>
+            <p>{truncate(goal.title.replace(/\s+/g, ""), 100)}</p>
```

Comment:
I am thinking 100 hundred is too much per your screenshot. Can you bump it down to 60?